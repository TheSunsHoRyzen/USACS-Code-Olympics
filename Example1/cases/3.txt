Lewis Math 88 English 92 Science 90
Sebastian Math 79 English 85 Science 88
Fernando English 88 Math 91
Micheal Science 90 English 95
Nico Math 90 English 75 