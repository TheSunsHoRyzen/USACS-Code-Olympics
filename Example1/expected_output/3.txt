Subject: Math
Average Grade: 87.0
Top Student(s): Fernando (91)

Subject: English
Average Grade: 87.0
Top Student(s): Micheal (95)

Subject: Science
Average Grade: 67.0
Top Student(s): Lewis, Micheal (90)
